vds_api_client/api_cli.py

```diff
@@ -286,6 +288,7 @@ def ts(ctx, config_file, products, latlons, rois, date_range, fmt,
                                  end_time=date_range[1] if date_range else None,
                                  lats=lats, lons=lons, rois=rois,
                                  av_win=av_win, av_win_dir=av_win_dir, masked=masked, clim=clim, t=t,
+                                 provide_coverage=coverage,
```

Comment:
For consistency with other arguments, I prefer `coverage=coverage`.

Reply:
then it should be `provide_coverage=provide_coverage` to be consisent with the api I guess. do you then also prefer `--provide_coverage`? It shortened it for less typing in the cli

Reply:
It's now all `provide_coverage`

Reply:
I see what you mean: `clim` and `t` are also shortened versions of the corresponding API parameters. If you prefer short, then `cov=cov` or `coverage=coverage` would also work for me.

Reply:
Yes, to harmonize those would be a different issue (and a new major version). For now what I will do is keep the `provide_coverage` intact and add a shortcut `-cov` to `--provide-coverage` so both options are available.

Reply:
Great, that works